Common.mk

```diff
@@ -448,13 +448,13 @@ $(REPO)/%ks-anywhere-go-mod-download: $(if $(PATCHES_DIR),$(GIT_PATCH_TARGET),$(
 ifneq ($(REPO),$(HELM_SOURCE_REPOSITORY))
 $(HELM_SOURCE_REPOSITORY):
 	git clone $(HELM_CLONE_URL) $(HELM_SOURCE_REPOSITORY)
+endif
```

Comment:
What was this causing?

Reply:
This may resolve when I move the helm stuff into release branch directories, but there was an issue where the HELM_GIT_TAG was not being checked out before helm patches were applied, causing errors.